backend/app/infrastructure/external_api/gc_passport/score.py

```diff
@@ -0,0 +1,74 @@
+import requests
+
+from app.constants import GC_PASSPORT_SCORER_API
+from app.exceptions import ExternalApiException
+from app.infrastructure.exception_handler import ExceptionHandler
+
+from app.settings import config
+
+
+def signing_message() -> str:
+    try:
+        response = requests.get(
+            f"{GC_PASSPORT_SCORER_API}/registry/signing-message",
+            headers=_authentication_headers(),
+        )
+        response.raise_for_status()
+        return response.json()
+
+    except requests.exceptions.RequestException as e:
+        ExceptionHandler.print_stacktrace(e)
+        raise ExternalApiException(e, 500)
+
+
+def issue_address_for_scoring(address: str) -> dict:
+    try:
+        response = requests.post(
+            f"{GC_PASSPORT_SCORER_API}/registry/submit-passport",
+            json={"address": address, "scorer_id": _scorer_id()},
+            headers=_authentication_headers(),
+        )
+        response.raise_for_status()
+        return response.json()
+
+    except requests.exceptions.RequestException as e:
+        ExceptionHandler.print_stacktrace(e)
+        raise ExternalApiException(e, 500)
+
+
+def fetch_score(address: str) -> dict:
+    try:
+        response = requests.get(
+            f"{GC_PASSPORT_SCORER_API}/registry/score/{_scorer_id()}/{address}",
+            headers=_authentication_headers(),
+        )
+        response.raise_for_status()
+        return response.json()
+
+    except requests.exceptions.RequestException as e:
+        ExceptionHandler.print_stacktrace(e)
+        raise ExternalApiException(e, 500)
+
+
+def fetch_stamps(address: str) -> dict:
+    try:
+        response = requests.get(
```

Comment:
please take a look how we use ```_get_api_url``` in other API-related modules and try to unify it

Reply:
:+1: